### ISSUE TITLE
Create a profile page

### ISSUE THREAD

**Issue by lucianomurr** [lucianomurr/angular-estrai.me#33]

**Comment by lucianomurr** [comment]
@AlessioMonterosso @mijorus 
could be useful to have a profile with all the historic information?

**Comment by AlessioMonterosso** [comment]
for profile page, do you mean a user profile page? if yes, historic information like game history?

**Comment by lucianomurr** [comment]
Yes, something related to the usage, game played, people registered, and more...
